features/charlie_creates_a_petition.feature

```diff
@@ -46,9 +47,37 @@ Scenario: Charlie creates our petition
   Then a petition should exist with title: "The wombats of wimbledon rock.", state: "validated"
   And there should be a "validated" signature with email "womboid@wimbledon.com" and name "Womboid Wibbledon"
 
+
+Scenario: Charlie creates a petition with valid postcode N1 1TY
+  Given I start a new petition
+  And ConstituencyAPI call N11TY has been stubbed
```

Comment:
We should probably handle the stubbing some other way. This reveals the implementation details which integration tests should not be concerned of. And the test cases should be understandable for non technical people too (this will probably make no sense for a non-programmer).